appsmithorg/appsmith
# fix: Add script to update MongoDB and Redis supervisor config when changing the URI

## Description

- The script will check for APPSMITH_MONGODB_URI and APPSMITH_REDIS_URL for **internal/external** and **enable/disable** supervisor config accordingly.
- Restart the related services including `backend, editor, rts`
![Flow](https://user-images.githubusercontent.com/72587752/158553135-3a6571b5-775b-4fdc-8ad6-8c69b4902632.png)

Fixes # (issue)
- [[Bug]: Embedded MongoDB not removed immediately when changing the MongoDB URI](https://github.com/appsmithorg/appsmith/issues/9967)

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?
- Have appsmith-ce running
- Update `APPSMITH_MONGODB_URI` ( using external src)
- Call `supervisor-mongodb.sh` script
- Check supervisord status.
## Checklist:
